cmd/main.go

```diff
@@ -702,14 +703,8 @@ func createStorageClient(deployTarget string, storage string, s3cfg *s3wrapper.C
 		switch storage {
 		case storage_s3:
 			storageClient = s3wrapper.NewS3Client(s3cfg, log)
-			if storageClient == nil {
```

Comment:
why were these removed

Reply:
the compiler indicated that the value of storageclient could never be nil and that these checks were not required.

Reply:
I can double check this and put them back if needed